flyway/postgre/migrations/sql/V1_3__Create_lesson2_tables.sql

```diff
@@ -1,69 +1,86 @@
-CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+DROP TYPE IF EXISTS organization_status_type;
```

Comment:
Thêm cái này mới dùng được hàm `uuid_generate_v4()`

```suggestion
CREATE EXTENSION IF NOT EXISTS "uuid-ossp";

DROP TYPE IF EXISTS organization_status_type;
```